skimage/transform/histogram_matching.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+
+
+def _get_separate_channels(image):
+    """Return an array consisting of all channels of an image"""
+    if len(image.shape) < 3:
+        return np.asarray([image])
+    else:
+        return image.transpose(2, 0, 1)
+
+
+def _match_array_values(source, template):
+    """Return modified source array with interpolated values that correspond to the
```

Comment:
Can this documentation be clarified?

Reply:
Renamed the function to `_match_cumulative_cdf`, reformulated the docstring.